Bottom: 37e9c258093ce3b2d806a14b80a9a02b538aeaef
Top:    ec81fd081f1d15175aef916baee3b625425853c3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-27 15:52:19 -0500

Refresh of tmp-update-openldap

---
diff --git a/roles/docker-openldap/tasks/main.yml b/roles/docker-openldap/tasks/main.yml
index 8a771906..a265c56a 100644
--- a/roles/docker-openldap/tasks/main.yml
+++ b/roles/docker-openldap/tasks/main.yml
@@ -167,12 +167,12 @@
 # *****************************************************************************
 # Set the LDAP passwords
 
-# - name: wait for LDAP to start
-#   wait_for:
-#     delay: 5
-#     host: '{{ openldap_ip_addr }}'
-#     port: 389
-#     state: started
+- name: wait for LDAP to start
+  wait_for:
+    delay: 5
+    host: '{{ openldap_ip_addr }}'
+    port: 389
+    state: started
 
 - name: set LDAP admin password
   command: 'docker exec -i {{ openldap_container_name }} /docker-entrypoint.sh set_admin_pass {{ openldap_admin_password | quote }}'
